components/brave_wallet/common/features.cc

```diff
@@ -47,6 +47,9 @@ BASE_FEATURE(kBraveWalletZCashFeature,
 #endif
 );
 
+const base::FeatureParam<bool> kZCashShieldedEnabled{
+    &kBraveWalletZCashFeature, "zcash_shielded_enabled", true};
```

Comment:
```suggestion
    &kBraveWalletZCashFeature, "zcash_shielded_enabled", base::FEATURE_DISABLED_BY_DEFAULT};
```

Reply:
probably needs linted still, but I'm fairly certain we want to leave this off by default for now

Reply:
Shielded what? Please don't sacrifice clarity for terseness. Also there should be a comment explaining what this flag does and the string should be camelcase, not underscore.

Reply:
> string should be camelcase, not underscore.

This is not a feature, this is a feature param, so camel case is not used

Reply:
Turned do default false, but it still under the buildflag check